master/reference/advanced/etcd-rbac/kubernetes.md

```diff
@@ -0,0 +1,85 @@
+---
+title: etcd RBAC with Kubernetes and Calico
+---
+
+When using etcd with RBAC, all components that access etcd must be configured
+with the proper certificates.  This document describes how to configure
+Kubernetes and Calico to access the same etcd cluster with etcd RBAC
+certificates.  This guide assumes you are following the general
+[Setting up etcd certificates for RBAC guide](index) and using its guidance
+for setting up certificates and etcd cluster, users, and roles.
+
+## Why you might be interested in this guide
+
+You are using Kubernetes and Calico who share an etcd datastore and wish
+to ensure that Calico and Kubernetes are unable to access each other's etcd
+data.
+
+## Needed etcd Roles
+
+The following components need certificates with a Common Name that matches an
+etcd user that has been given appropriate roles allowing access to the key
+prefixes or paths listed below.
+- kube-apiserver
+  - Read and Write access to `/registry`.
+    - With Kubernetes v1.6 the etcd v3 API is used, this can be switched
+	  to etcd v2 by passing a flag to kube-apiserver.
+	- This key prefix can be changed by passing the proper option to the
+	  kube-apiserver.
+  - The etcd user needs to be given the root role to perform compaction.
```

Comment:
Does the point about requiring root permissions make the point about read/write access to /registry moot?